data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,426 @@ mission "Remnant: Face to Maw 2B"
+			`	Aeolus looks faintly amused. "While you are certainly adventurous, you are not the only one to have traveled widely. We mapped the Hai systems long ago with cloaked ships, but never landed or contacted any of them. So we never knew anything more than what we could observe from orbit or from scanning ships. But you are known to them, and have a fairly public reputation. By using your transponder codes, we can place and pay for orders through hyperspace relays. All we need is a ship in position to feed the signal into the network.`
+				accept
+			label remnantuntrusted
+			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
+				accept
+	on complete
+		dialog
+			`It takes a few minutes to contact the yardmaster, but once you do, a mixed crew of Hai and Human dockworkers have the crates moved promptly from the warehouse to your cargo bay. Hai and Humans seem accustomed to working together here.`
+
+
+
+mission "Remnant: Crystal Research 2"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
+	landing
+	source "Greenwater"
+	destination "Allhome"
+	cargo "Keystones" 2
+	to offer
+		has "Remnant: Crystal Research 1: done"
+	on complete
+		payment 250000
+
+
+
+mission "Remnant: Crystal Research 3"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from several Hai worlds, ending on Newhome.`
+	landing
+	source "Allhome"
+	destination "Newhome"
+	stopover "Makerplace"
+	stopover "Stonebreak"
+	stopover "Giverstone"
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Icelake"
+	stopover "Heartvalley"
+	cargo "Keystones" 4
+	to offer
+		has "Remnant: Crystal Research 2: done"
+	on offer
+		conversation
+			`True to Aeolus' word, there are crates of Hai Quantum Keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message has arrived. It has no indication of provenance, but appears to be a notification that a payment of 250,000 marked 'on-assignment allowance' has been made to your account.`
+			`	The attached message is brief and to the point. "Greetings, Captain <first>. Hopefully the shipments were waiting for you. We have made arrangements for samples to be acquired from Makerplace, Stonebreak, Giverstone, Cloudfire, Snowfeather, Icelake, and Heartvalley. Once you have collected all those, head to Newhome. There are a few other places that we are still working to set up, and we should have those ready by the time you reach Newhome."`
+				accept
+	on stopover
+		dialog
+			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to provide the smoothest flight for your passengers!" reads one of the labels.`
```

Comment:
Every appliance or house fixture I've ever owned has come with brochures advertising more of whatever it was I bought and talking about how good a purchase it was. I mean, really. It is a stove. If I haven't managed to recycle this brochure by the time this one needs to be replaced, then chances are I'm not going to trust the same brand a second time. And yet, it still happens. Maybe they're hoping that I'll have such a good experience with the new stove that I give the brochure to my friends?

I included it because *I* notice things like whether or not brands are listed on the package, so I assume that my pilot will, too. I based a fair bit of this on the description of the keystone, which states:

> "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

To me, that sounds like prime grounds for advertising that this or that brand makes a better version. Either that, or there is a monopolistic single supplier whose product over the millenia has reached religious-symbol level of devotion. And while we haven't really seen much or anything of Hai spiritual beliefs, they do have strong "Just like humanity, except better" traits. Hence the advertising descriptions are intended to sound like they are targeted at either star liner captains (as in, captains that cater to rich/upper class passengers), or well-off people themselves. Given what we know of the Hai and the seemingly intended "just like humanity but better" feel, having a well-functioning capitalistic society with a whole variety of of suppliers makes sense. 

Worth noting that - Remnant attitudes aside - the price of a Keystone versus the price of a Key Stone indicates that the Remnant superiority might not be as clearly superior as they would like to think.